Transfer/Sources/ScheduleTransferController.swift

```diff
@@ -201,10 +206,18 @@ extension ScheduleTransferController: ScheduleTransferView {
     }
 
     func showConfirmation(handler: @escaping (() -> Void)) {
+        let destinationData = presenter!.sectionData[0] as? ScheduleTransferDestinationData
         processingView?.hide(with: .complete)
```

Comment:
There's no processing View now, Just show acitivityIndicator during the processing and then Confirmation Alert